implementations/rust/ockam/ockam_api/src/nodes/service.rs

```diff
@@ -168,14 +251,22 @@ impl NodeMan {
                     self.transports.len() as u32,
                 ))
                 .encode(enc)?,
-            // == Get all transports
+
+            // ==*== Transports ==*==
+
+            // TODO: Get all transports
             (Get, "/node/transport") => Response::ok(req.id())
                 .body(TransportList::new(self.get_transports()))
                 .encode(enc)?,
-            // TODO: Get all transports
-            // == Create a new transport
             (Post, "/node/transport") => self.add_transport(req, dec).await?.encode(enc)?,
             (Delete, "/node/transport") => self.delete_transport(req, dec).await?.encode(enc)?,
+
+            // ==*== Inlets & Outlets ==*==
+            (Get, "/node/portal") => self.get_iolets(req).encode(enc)?,
+            (Post, "/node/portal") => self.create_iolet(ctx, req, dec).await?.encode(enc)?,
```

Comment:
I think having 2 separate endpoints would be convenient, since arguments are different
Also, we should stick to either portal or iolet term